tests/common/params_test.py

```diff
@@ -14,3 +14,15 @@ def test_load_from_file(self):
 
         model_params = params.pop("model")
         assert model_params.pop("type") == "bidaf"
+
+    def test_overrides(self):
+        filename = 'tests/fixtures/bidaf/experiment.json'
+        params = Params.from_file(filename, '{ "train_data_path": "FOO", "model": { "type": "BAR" }, model.text_field_embedder.tokens.type: "BAZ" }') #pylint: disable=line-too-long
```

Comment:
Instead of disabling pylint, can you just move the string to its own line, so it's isn't too long?

Reply:
whitespace is implicitly string concatenation, so you should also be able to do something like

```
params = Params.from_file(filename, 
                          '{ "train_data_path": "FOO",'
                          '  "model": { "type": "BAR" },' 
                          '  model.text_field_embedder.tokens.type: "BAZ" }')')
```

if you find that tolerable